ingestion/Dockerfile.ci

```
@@ -82,7 +82,7 @@ WORKDIR /home/airflow/openmetadata-airflow-apis
 RUN pip install "."
 
 WORKDIR /home/airflow/ingestion
-ARG INGESTION_DEPENDENCY="all"
+ARG INGESTION_DEPENDENCY
```

is this change intended?

---

```suggestion
ARG INGESTION_DEPENDENCY="all"
```

I am reverting it. We can open another PR if we need this in